srcalc/src/main/java/gov/va/med/srcalc/domain/model/AbstractVariable.java

```diff
@@ -60,52 +73,58 @@ public int getId()
      * For reflection-based construction only. Business code should never modify
      * the surrogate key as it is generated from the database.
      */
-    void setId(final int id)
+    final void setId(final int id)
```

Comment:
I didn't think about it before, but it makes sense that the common methods won't need to be overridden.